vpd_tool_impl.cpp

```diff
@@ -989,49 +989,52 @@ void VpdTool::parseSVPDOptions(const nlohmann::json& json)
 
                         cin >> option;
                         cout << '\n' << outline << endl;
-
                         EditorImpl edit(constants::systemVpdFilePath, json,
                                         get<1>(data), get<2>(data));
-                        string value;
-
-                        switch (option)
+                        if (option == VpdTool::BMC_DATA_FOR_CURRENT)
```

Comment:
can you change this to BACKUP_DATA_FOR_CURRENT
and instead of BMC_DATA_FOR_ALL change it to BACKUP_DATA_FOR_ALL

Reply:
It won't have effect on the display, so may not be required, I think. As it's downstream code and variables have been approved before.

Reply:
this change is required to have the naming convention consistent all over the code

Reply:
Okay, I'll change but not sure though. Thanks.